pages/events/_id/dashboard.vue

```diff
@@ -88,7 +88,7 @@
                     update(item.partnerId, { partnerId: '' })
                     update(item.id, { partnerId: '' })
                   "
-                  >Unlink</TButton
+                  >{{$t("EventDashboard.buttonUnlink  ")}}</TButton
```

Comment:
Did you test it? This key won't be found as you have extra spaced here.

```suggestion
                  >{{$t("EventDashboard.buttonUnlink")}}</TButton
```

Reply:
Okay, I'll see to it